corehq/apps/users/forms.py

```diff
@@ -1169,7 +1169,6 @@ class UserFilterForm(forms.Form):
         required=False,
         label=_('Project Spaces'),
         widget=forms.SelectMultiple(attrs={'class': 'hqwebapp-select2'}),
-        help_text=_('Add project spaces containing the desired users'),
```

Comment:
Ya, that help text seemed a bit too much but was thinking if it was added for a good reason.
As a rephrase, I was thinking "Add project spaces to get users from" just to make it explicit.